stix_shifter_modules/gcp_chronicle/configuration/config.json

```diff
@@ -1,7 +1,8 @@
 {
 	"connection": {
 		"type": {
-			"displayName": "Google Chronicle Security"
+			"displayName": "Google Chronicle Security",
+			"group": "chronicle"
```

Comment:
should we call it gcp? since more gcp connectors might be coming?

Reply:
Yeah, I'll change it.